packages/components/CLAUDE.md

```
@@ -0,0 +1,297 @@
+
+```tsx
+// ✅ OK - single level import
+import { TextFilter } from '@redhat-cloud-services/frontend-components/ConditionalFilter';
+
+// ❌ Wrong - breaks module resolution
+import TextFilter from '@redhat-cloud-services/frontend-components/ConditionalFilter/TextFilter';
+```
+
+## Component Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)